pkg/api/routers/fileops.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2019 OpenSDS Authors All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package routers
+
+import (
+	"github.com/astaxie/beego"
+	"github.com/opensds/opensds/pkg/api/controllers"
+	"github.com/opensds/opensds/pkg/utils/constants"
+)
+
+func init() {
+
+	// add router for file ops api
+	filens :=
+		beego.NewNamespace("/"+constants.APIVersion,
+
+			// All operations of file can be used for both admin and users.
+			beego.NSRouter("/uploadconf", controllers.NewFileOpsPortal(), "post:UploadConfFile"),
+			beego.NSRouter("/downloadconf", controllers.NewFileOpsPortal(), "get:DownloadConfFile"),
```

Comment:
Why do we need APIs for uploading/downloading config files?

Reply:
This was added to add functionality to telemetry after the initial installation. Some use cases are

- add a new exporter (CEPH) to Prometheus 
- define new threshold rules in Prometheus to raise Alerts to Alert manager
etc. 
The primary driver for adding these API was that during a demo (in a meetup, conference etc.) we do NOT need to keep switching between the UI and the backend console to do these configuration changes (either by running a script or by directly editing the configuration files)

Reply:
I don't think it's a good idea to add a new first-level API, if this is a required operation, please move it under metrics path.

Reply:
added subpath ```metrics/```